index.js

```diff
@@ -1,4 +1,7 @@
 function fetchBooks() {
+  return fetch('https://anapioficeandfire.com/api/books')
+  .then(resp => resp.json())
+  .then(renderBooks)
```

Comment:
@thuyanduong-flatiron Is this the solution code? Should this be removed?

Reply:
I complete didn't notice. I have removed it now.